packages/graphql/src/graphql/objects/UpdateInfo.ts

```diff
@@ -21,6 +21,7 @@ import { GraphQLInt, GraphQLNonNull, GraphQLObjectType, GraphQLString } from "gr
 
 export const UpdateInfo = new GraphQLObjectType({
     name: "UpdateInfo",
+    description: "Information about the update of a node or relationship.",
```

Comment:
I think a more accurate description would be "Information about the number of nodes and relationships created and deleted during an update mutation".